store.js

```diff
@@ -0,0 +1,36 @@
+const electron = require('electron')
+const path = require('path')
+const fs = require('fs')
+
+class Store {
+  constructor (opts) {
+    const isProduction = process.env.NODE_ENV !== 'development'
+    // Use a different userData file for development
```

Comment:
i think you still need to do app.setPath here; otherwise prod and dev will share the same app data directory.